code/css-grid/styles.css

```diff
@@ -0,0 +1,38 @@
+#wrapper {
+  font-size: 2em;
+  display: grid;
+  grid-template-columns: 10rem 3fr 1fr;
+  grid-template-rows: min-content 1fr min-content;
```

Comment:
In this case, every row has at least one element with an explicitly defined height. These override the row sizes you defined here. If you take this line out it won't change anything. :)